lib/routemaster/middleware/metrics.rb

```diff
@@ -22,6 +22,9 @@ def call(request_env)
           rescue Routemaster::Errors::BaseError => e
             increment_response_count(response_tags(e.env))
             raise e
+          rescue Faraday::TimeoutError => e
```

Comment:
Should we add connection too?

Reply:
Wdym? `Routemaster::Errors::BaseError` and `Faraday::TimeoutError` are the only exceptions that are monitored by CircuitBox

Reply:
`Faraday::ConnectionFailed`